vm/src/obj/objint.rs

```diff
@@ -369,12 +377,18 @@ impl PyInt {
     #[pymethod(name = "__mod__")]
     fn mod_(&self, other: PyObjectRef, vm: &VirtualMachine) -> PyResult {
         if objtype::isinstance(&other, &vm.ctx.int_type()) {
-            let v2 = get_value(&other);
-            if *v2 != BigInt::zero() {
-                Ok(vm.ctx.new_int((&self.value) % v2))
-            } else {
-                Err(vm.new_zero_division_error("integer modulo by zero".to_string()))
-            }
+            let other = other.payload::<PyInt>().unwrap();
+            inner_mod(self, &other, vm)
+        } else {
+            Ok(vm.ctx.not_implemented())
+        }
+    }
+
+    #[pymethod(name = "__rmod__")]
+    fn rmod_(&self, other: PyObjectRef, vm: &VirtualMachine) -> PyResult {
```

Comment:
`mod` is a Rust keyword, but `rmod` is not. You can just use `rmod` instead of `rmod_`.

Reply:
Thank you